catalog/app/utils/Config.ts

```diff
@@ -90,6 +91,7 @@ const transformConfig = (cfg: ConfigJson) => ({
   apiGatewayEndpoint: startWithOrigin(cfg.apiGatewayEndpoint),
   noDownload: !!cfg.noDownload,
   noOverviewImages: !!cfg.noOverviewImages,
+  /** @deprecated */
   desktop: !!cfg.desktop,
```

Comment:
you can remove it from here if it's not used in the code

Reply:
It is used. I didn't want to clutter that PR with removing the use of `cfg.desktop` in components unrelated to the PR.